fastpass/src/main/scala/scala/meta/internal/fastpass/bazelbuild/Bazel.scala

```diff
@@ -449,4 +464,36 @@ class Bazel(bazelPath: Path, cwd: Path) {
       .exitValue()
   }
 
+  def runWithErrorCatching(
+      cmd: List[String],
+      out: ByteArrayOutputStream,
+      err: OutputStream = FileUtils.NullOutputStream,
+      exc: String
+  ): Int = {
+    val errByteArrayOutputStream = new ByteArrayOutputStream
+    val io =
+      new ProcessIO(
+        _ => (),
+        FileUtils.copy(_, out),
+        FileUtils.copy(_, errByteArrayOutputStream)
+      )
+
+    val errWriter = new PrintWriter(err)
+    errWriter.println(s"# Running command in ${cwd}:")
+    errWriter.println(s"$$ ${cmd.mkString(" ")}")
+    val exitCode = Process(cmd, cwd.toFile)
```

Comment:
Instead of inlining the definition of `Bazel.run` here, is it possible to call it instead?